CRM/Admin/Page/MessageTemplates.php

```diff
@@ -199,9 +199,11 @@ public function run($args = NULL, $pageArgs = NULL, $sort = NULL) {
 
       CRM_Core_BAO_MessageTemplate::revert($id);
     }
-
-    $this->assign('selectedChild', CRM_Utils_Request::retrieve('selectedChild', 'String', $this));
-
+    $selectedChild = CRM_Utils_Request::retrieve('selectedChild', 'String', $this);
+    $acceptableChildren = array('user', 'workflow');
+    if (in_array($selectedChild, $acceptableChildren)) {
```

Comment:
Just a minor change. Since ```$acceptableChildren``` is not used anywhere other then this IF condition so I think we should change it to 
```php
- $acceptableChildren = array('user', 'workflow');
-  if (in_array($selectedChild, $acceptableChildren)) {
+ if (in_array($selectedChild, array('user', 'workflow'))) {
```

Reply:
thanks @monishdeb have updated it now